tests/css-shadow-parts-1.js

```diff
@@ -13,7 +13,60 @@ export default {
 				tr: '#part',
 				dev: '#part',
 			},
-			tests: ['::part(label)'],
+			tests: [
+				'::part(label)',
+				'::part(mypart):active',
+				'::part(mypart):any-link',
+				'::part(mypart):autofill',
+				'::part(mypart):checked',
+				'::part(mypart):closed',
+				'::part(mypart):default',
+				'::part(mypart):defined',
+				'::part(mypart):dir(ltr)',
+				'::part(mypart):disabled',
+				'::part(mypart):enabled',
+				'::part(mypart):focus',
+				'::part(mypart):focus-visible',
+				'::part(mypart):focus-within',
+				'::part(mypart):fullscreen',
+				'::part(mypart):future',
+				'::part(mypart):hover',
+				'::part(mypart):indeterminate',
+				'::part(mypart):in-range',
+				'::part(mypart):invalid',
+				'::part(mypart):lang(en)',
+				'::part(mypart):link',
+				'::part(mypart):modal',
+				'::part(mypart):open',
+				'::part(mypart):optional',
+				'::part(mypart):out-of-range',
+				'::part(mypart):past',
+				'::part(mypart):paused',
+				'::part(mypart):picture-in-picture',
+				'::part(mypart):placeholder-shown',
+				'::part(mypart):playing',
+				'::part(mypart):popover-open',
+				'::part(mypart):read-only',
+				'::part(mypart):read-write',
+				'::part(mypart):required',
+				'::part(mypart):state(mystate)',
+				'::part(mypart):target',
+				'::part(mypart):user-invalid',
+				'::part(mypart):user-valid',
+				'::part(mypart):valid',
+				'::part(mypart):visited',
+				'::part(mypart):xr-overlay',
+
+				'::part(mypart):target-within',
+				'::part(mypart):local-link',
+				'::part(mypart):seeking',
+				'::part(mypart):buffering',
+				'::part(mypart):stalled',
+				'::part(mypart):muted',
+				'::part(mypart):volume-locked',
```

Comment:
@dbaron I noticed that these are not currently added to the tests in Blink Cl, are these valid?

Reply:
It's in the spec for the selectors:
https://drafts.csswg.org/selectors-4/#sound-state

Reply:
As I interpret them, `:seeking`, `:buffering`, `:stalled`, `:muted`, and `:volume-locked` fulfill the criteria for being placed after `::part()`, as they match `<audio>` and `<video>` elements based on their local element information. (And David at least already added `:paused` and `:playing`.)

`:current` (and its functional notation), `:future`, and `:past` on the other side, not, because they are structural pseudo-classes, i.e. they match an element based on another element.

Reply:
:future, and :past on the other side, not, because they are structural pseudo-classes, i.e. they match an element based on another element.

Can you go here and make a comment? https://github.com/w3c/csswg-drafts/issues/10787

Reply:
I've [done that now](https://github.com/w3c/csswg-drafts/issues/10787#issuecomment-2474945032). Sorry for the long delay!

Sebastian